### ISSUE TITLE
Warp Drive exotic matter charge amount

### ISSUE THREAD

**Issue by embermctillhawk** [sswelm/KSP-Interstellar-Extended#560]
Pressing the "start charging" button will cause the exotic matter amount to flicker around the needed amount to initiate warp. The only way to be sure that warp will actually initiate when pushing the button is to manually specify 110% exotic matter or greater.

**Comment by sswelm** [comment]
could you give any example under what condition this happens?

**Comment by embermctillhawk** [comment]
With further experimentation, I have determined the condition where this happens.

This happens when having Near Future Electric installed.

Ascending/Descending doesn't appear to have any effect, nothing is overheating or shutting down, we're totally fine on power.

A vehicle which is doing this can be saved, then the game closed, NFE uninstalled, then opened again, and the flickering is gone.

As of writing I have yet to verify that re-adding NFE will bring the flickering back but I believe it is a safe assumption.

**Comment by sswelm** [comment]
What about the latest release where I attempted to improve warp charging stability?

**Comment by embermctillhawk** [comment]
I'm not entirely certain on where to find the latest release. If it's on Curseforge, then I have tried it, and the behavior is still the same.

It might be a general issue with NFE, because I also note that on the test vehicle I built, with NFE, if I attempt to activate the QSR while in low kerbin orbit, I get a message that I don't have enough power, but it slowly charges from the batteries, then activates. Without NFE, it simply gives the power required/reduce altitude message and does nothing interesting.

Also I can activate the QSR instantly anyways by choosing "Activate" then "Next Fusion Mode".
